lm_eval/api/model.py

```diff
@@ -312,7 +317,7 @@ def loglikelihood(
             if context == "":
                 # end of text as context
```

Comment:
this comment is confusing. proposed change allows BOS token (begin of sequence) instead of empty context. May this be changed?